vmaas_sync/vmaas_sync.py

```diff
@@ -67,6 +68,7 @@ def sync_cve_md(webhook_queue, page_size=5000):  # pylint: disable=too-many-bran
                 LOGGER.info('Importing CVE metadata (page: %s, page_size: %s, pages: %s)',
                             page, page_size, r_json['pages'])
                 cves = r_json['cve_list']
+                all_rh_cves += [cve for cve in cves]
```

Comment:
`all_rh_cves += [cve for cve in cves]`

why not `all_rh_cves += cves` or `all_rh_cves.extend(cves)`?

Reply:
I'm thinking that it will be fewer strings in the list then just pushing dict into it